### ISSUE TITLE
how to change string encoding charset

### ISSUE THREAD

**Issue by zhengdi1992** [bytedeco/javacpp#293]
for history reason..i can not change the global setting to change `file.encoding=UTF8`
maybe i will cause some problem unpredictably..
but now i need to load a .pb file and input is a string tensor..but my app's file encoding default was `US-ASCII`  print by `System.out.println(Charset.defaultCharset())`

i want to know whether exist api for change the charset encoding in small scope or not?
`tensorflow.Tensor mockInput = new tensorflow.Tensor(tensorflow.DT_STRING, new TensorShape(new long[]{1}));`
`mockInput.createStringArray().put("香蕉 苹果 西瓜");` 

such as : 
`mockInput.createStringArray().put("香蕉 苹果 西瓜","UTF-8");` ?

i have try this but not worked.. :(
`mockInput.tensor_data().putString("香蕉 苹果 西瓜","UTF-8");`

**Comment by saudet** [comment]
Oops, sorry about that. I've added a `StringArray.putString(BytePointer)` method to fix this issue in commit https://github.com/bytedeco/javacpp-presets/commit/5e3bf2a6ba59387c5928da3059261b25e05b92e0, so we can now do something like:
`createStringArray().putString(new BytePointer("香蕉 苹果 西瓜", "UTF-8"))`.
Please give it a try with the snapshots: http://bytedeco.org/builds/

**Comment by zhengdi1992** [comment]
thx. i've see the push to master
and i've use
```
     <dependency>
        <groupId>org.bytedeco.javacpp-presets</groupId>
        <artifactId>tensorflow</artifactId>
        <version>1.5.0-1.4</version>
      </dependency>
      <dependency>
        <groupId>org.bytedeco.javacpp-presets</groupId>
        <artifactId>tensorflow</artifactId>
        <version>1.5.0-1.4</version>
        <classifier>macosx-x86_64</classifier>
      </dependency>
```
was i should change the version to which ?.snapshot


and additionally..
any method for like `createStringArray().toString("UTF-8")`

**Comment by saudet** [comment]
See the version here:
https://github.com/bytedeco/javacpp-presets/tree/master/tensorflow

For the other way around, we can use: data().capacity(size()).getString("UTF-8")

**Comment by saudet** [comment]
The changes have now been included with version 1.5. Please let me know if you encounter any other issues and thanks for reporting!
